pkg/mcp/creds/watcher.go

```diff
@@ -43,7 +43,7 @@ var scope = log.RegisterScope("mcp", "mcp debugging", 0)
 
 const (
 	// defaultCertDir is the default directory in which MCP options reside.
-	defaultCertDir = "/etc/istio/certs/"
+	defaultCertDir = "/etc/certs/"
```

Comment:
?
can you undo this please so that this PR is scoped

Reply:
Ok.